### ISSUE TITLE
Colorblindness Accessibility

### ISSUE THREAD

**Issue by jtravis-nj** [Reviewable/Reviewable#1082]
Accessibility options for colorblind people seems to be lacking.  To mitigate this, there should be an option in the reviewable settings to do the following:
1. Change the diff highlight colors
2. Change the color of the "eyes" in reviewable (for instance, in the file matrix)

Needing to modify and make available some css document is a barrier for entry for accessibility options.

**Comment by pkaminski** [comment]
Agreed -- we'll add a "color blind scheme" toggle to the account settings in the near future.  We'll need to do some non-trivial internal refactoring first, though.

**Comment by earlAchromatic** [comment]
There is now a color replacement toggle in account settings that lets you replace red/green with custom colors throughout Reviewable.
